truffle-config.js

```diff
@@ -0,0 +1,152 @@
+require("dotenv").config();
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+
+const CONSISTENT_DEPOLY_ADDR = "0xB3c12C9535626cfFef5FE76d5A4e198A5e753FD9";
+// const CONSISTENT_DEPOLY_ADDR = "0x70ebDF2a1C9C7A45141760419c42056E3DDfA690";
+// const CONSISTENT_DEPOLY_ADDR ="0x1Dde3A77975235E6DdB31389c3D599dE67AbA1Ad";
+module.exports = {
+  // See <http://truffleframework.com/docs/advanced/configuration>
+  // to customize your Truffle configuration!
+  networks: {
+    // mainnet config
+    ethereum: {
+      provider: new HDWalletProvider(
+        process.env.SAME_ADDR_KEY,
```

Comment:
nit: use the same
```
const {.....} = process.env;
```
to have an overview of the supported environment variables.

Reply:
thanks, using 
const {SAME_ADDR_KEY, SAME_ADDR, ETHEREUM_URL, SEPOLIA_URL,
      ETHERSCAN_API_KEY, POLYGONSCAN_API_KEY, BSCSCAN_API_KEY, SNOWTRACE_API_KEY} = process.env;